### DISCUSSION TITLE
Pandas tutorials

### DISCUSSION THREAD

**Discussion by harrylloyd-bl** [Digital-Preservation-Coalition/PythonStudyGroups#14] [Recommended Resources]
One study group was asking about [pandas](https://pandas.pydata.org/) for data analysis and any good resources. It's a very useful package that you might consider if:
- you're doing data analysis
- you thought about doing it in excel
- you have delimited (.csv/.tsv) data
- you have a bunch of lists and dicts and it's all getting a bit much to keep track of

A few resources:
- From the Pandas docs [Getting started](https://pandas.pydata.org/docs/getting_started/index.html) and [10 mins to pandas](https://pandas.pydata.org/docs/user_guide/10min.html)
- Video series from [Python Tutorials for Digital Humanities](https://www.youtube.com/playlist?list=PL2VXyKi-KpYsjfozVCczTi2GxNJS5dKTD)
- Written lesson on [Programming Historian](https://programminghistorian.org/en/lessons/crowdsourced-data-normalization-with-pandas)
- Another written option on the [Python for Humanities](https://carpentries-incubator.github.io/python-humanities-lesson/) carpentry
- Dive straight in with this [Berkely colab notebook](https://ischoolonline.berkeley.edu/blog/python-pandas-practice-problems/ ) with a few intro problems

The Youtube channel and Programming Historian have a wealth of other digital humanities content appropriate to digital preservation as well!

**Comment by mpstrickland** [comment]
[like]  Manuela Pallotto Strickland reacted to your message:
________________________________
From: harrylloyd-bl ***@***.***>
Sent: Wednesday, January 24, 2024 4:38:53 PM
To: Digital-Preservation-Coalition/PythonStudyGroups ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Digital-Preservation-Coalition/PythonStudyGroups] Pandas tutorials (Discussion #14)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

One study group was asking about pandas<https://pandas.pydata.org/> for data analysis and any good resources. It's a very useful package that you might consider if:

  *   you're doing data analysis
  *   you thought about doing it in excel
  *   you have delimited (.csv/.tsv) data
  *   you have a bunch of lists and dicts and it's all getting a bit much to keep track of

A few resources:

  *   From the Pandas docs Getting started<https://pandas.pydata.org/docs/getting_started/index.html> and 10 mins to pandas<https://pandas.pydata.org/docs/user_guide/10min.html>
  *   Video series from Python Tutorials for Digital Humanities<https://www.youtube.com/playlist?list=PL2VXyKi-KpYsjfozVCczTi2GxNJS5dKTD>
  *   Written lesson on Programming Historian<https://programminghistorian.org/en/lessons/crowdsourced-data-normalization-with-pandas>
  *   Dive straight in with this Berkely colab notebook<https://ischoolonline.berkeley.edu/blog/python-pandas-practice-problems/> with a few intro problems

The Youtube channel and Programming Historian have a wealth of other digital humanities content appropriate to digital preservation as well!
